2024-12-23 04:00:16,873 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f629b65fd30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2024-12-23 04:00:16,873 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f629b65fd30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2024-12-31 17:00:37,448 - ERROR - ETL Process for Workly failed: APIError: [503]: The service is currently unavailable.
2025-01-01 14:00:03,309 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f4d0c11f9a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-01-01 14:00:03,312 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f4d0c11f9a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-01-04 10:00:50,906 - ERROR - ETL Process for Workly failed: APIError: [503]: The service is currently unavailable.
2025-01-05 10:00:40,055 - ERROR - ETL Process for Workly failed: APIError: [503]: The service is currently unavailable.
2025-01-06 02:00:14,023 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe97e217d60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 02:00:14,024 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe97e217d60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 03:00:14,281 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f1910003940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 03:00:14,291 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f1910003940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 07:00:14,179 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc42e7bb310>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 07:00:14,183 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc42e7bb310>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 14:00:12,805 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5e8e8b3550>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 14:00:12,815 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5e8e8b3550>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 17:00:13,076 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f785389f760>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 17:00:13,078 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f785389f760>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 19:00:12,954 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe8aa56b340>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 19:00:12,956 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe8aa56b340>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 20:00:14,081 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7feaf9aabd90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-06 20:00:14,084 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7feaf9aabd90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 03:00:15,066 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe68027fbb0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 03:00:15,069 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe68027fbb0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 04:00:16,361 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f866069b700>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 04:00:16,369 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f866069b700>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 05:00:13,578 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f6825737ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 05:00:13,580 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f6825737ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 06:00:12,804 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2be3dfbb80>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 06:00:12,806 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2be3dfbb80>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 07:00:04,760 - ERROR - Request failed for page 1: 402 Client Error: Payment Required for url: https://api.workly.uz/v1/reports/inouts?start_date=2024-11-01&page=1&per-page=50
2025-01-07 07:00:04,761 - ERROR - ETL Process for Workly failed: Failed to fetch inouts for page 1: 402 Client Error: Payment Required for url: https://api.workly.uz/v1/reports/inouts?start_date=2024-11-01&page=1&per-page=50
2025-01-07 08:00:14,501 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f98aabfbb50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 08:00:14,503 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f98aabfbb50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 11:00:44,002 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 12:00:38,990 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 13:00:13,312 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f00c50531f0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 13:00:13,313 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f00c50531f0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-07 14:00:39,480 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 15:00:38,898 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 16:00:40,226 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 17:00:41,752 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 18:00:45,932 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 19:00:40,527 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 20:00:44,686 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 21:00:39,563 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 22:00:45,604 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-07 23:00:40,362 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 00:00:39,156 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 01:00:44,218 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 02:00:14,295 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f44a630f970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 02:00:14,297 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f44a630f970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 03:00:44,716 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 04:00:16,066 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f8f38f9f520>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 04:00:16,070 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f8f38f9f520>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 05:00:13,899 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb718967b20>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 05:00:13,900 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb718967b20>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 06:00:44,147 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 07:00:43,814 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 08:00:14,118 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f168a52b670>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 08:00:14,120 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f168a52b670>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 09:00:45,225 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 10:00:14,397 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0563e83a30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 10:00:14,400 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0563e83a30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-08 11:00:45,012 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 12:00:45,244 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 13:00:39,612 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 14:00:44,586 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 15:00:45,468 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 16:00:40,411 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 17:00:42,806 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 18:00:39,686 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 19:00:40,282 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 20:00:41,296 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 21:00:41,227 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 22:00:40,579 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-08 23:00:40,124 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 00:00:41,329 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 01:00:39,809 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 02:00:40,858 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 03:00:39,583 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 04:00:41,838 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 05:00:41,913 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 06:00:40,983 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 07:00:40,451 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 08:00:41,487 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 09:00:42,607 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 10:00:41,374 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 11:00:41,238 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 12:00:40,619 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 13:00:40,530 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 14:00:40,695 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 15:00:41,702 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 16:00:40,250 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 17:00:44,258 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 18:00:41,568 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 19:00:41,539 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 20:00:39,803 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 21:00:42,225 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 22:00:43,178 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-09 23:00:42,231 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 00:00:41,280 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 01:00:40,671 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 02:00:41,730 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 03:00:40,581 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 04:00:44,180 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 05:00:44,031 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 06:00:40,628 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 07:00:41,757 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 08:00:43,226 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 09:00:44,135 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 10:00:44,455 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 11:00:44,728 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 12:00:41,732 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 13:00:42,764 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 14:00:42,159 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 15:00:42,043 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 16:00:43,056 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 17:00:44,085 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 18:00:43,277 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 19:00:43,150 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 20:00:41,806 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 21:00:41,578 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 22:00:42,985 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-10 23:00:42,867 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 00:00:42,844 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 01:00:43,453 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 02:00:42,189 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 03:00:42,878 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 04:00:47,449 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 05:00:44,670 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 06:00:42,688 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 07:00:44,313 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 08:00:43,649 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 09:00:45,213 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 10:00:44,285 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 11:00:42,141 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 12:00:43,291 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 13:00:40,909 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 14:00:42,171 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 15:00:41,739 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 16:00:41,966 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 17:00:42,784 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 18:00:42,568 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 19:00:41,972 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 20:00:41,587 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 21:00:42,321 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 22:00:41,391 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-11 23:00:41,159 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 00:00:42,942 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 01:00:40,440 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 02:00:41,303 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 03:00:41,099 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 04:00:46,193 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 05:00:41,645 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 06:00:41,129 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 07:00:40,298 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 08:00:46,038 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 09:00:40,004 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 10:00:41,213 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 11:00:41,865 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 12:00:41,182 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 13:00:40,814 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 14:00:40,370 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 15:00:40,759 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 16:00:41,395 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 17:00:44,389 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 18:00:42,057 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 19:00:41,511 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 20:00:41,982 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 21:00:40,782 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 22:00:42,626 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-12 23:00:41,765 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 00:00:41,772 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 01:00:41,158 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 02:00:41,907 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 03:00:41,632 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 04:00:43,709 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 05:00:42,624 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 06:00:42,378 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 07:00:41,593 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 08:00:51,514 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 09:00:43,907 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 10:00:43,204 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 11:00:43,810 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 12:00:44,397 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 13:00:43,644 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 14:00:43,548 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 15:00:44,084 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 16:00:43,472 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 17:00:45,317 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 18:00:44,199 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 19:00:42,250 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 20:00:42,547 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 21:00:43,403 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 22:00:44,143 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-13 23:00:44,401 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 00:00:41,970 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 01:00:42,231 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 02:00:42,051 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 03:00:41,702 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 04:00:44,575 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 05:00:43,418 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 06:00:41,815 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 07:00:42,875 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 08:00:43,388 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 09:00:43,620 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 10:00:43,932 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 11:00:42,453 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 12:00:43,464 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 13:00:43,083 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 14:00:42,679 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 15:00:42,615 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 16:00:44,261 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 17:00:43,974 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 18:00:44,780 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 19:00:43,534 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 20:00:43,743 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 21:00:42,636 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 22:00:44,261 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-14 23:00:44,751 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 00:00:45,258 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 01:00:43,598 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 02:00:43,683 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 03:00:44,006 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 04:00:46,278 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 05:00:48,074 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 06:00:42,493 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 07:00:43,142 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 08:00:46,269 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 09:00:49,854 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 10:00:50,306 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 11:00:44,681 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 12:00:43,951 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 13:00:43,525 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 14:00:43,174 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 15:00:44,255 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 16:00:46,143 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 17:00:45,816 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 18:00:47,338 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 19:00:44,343 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 20:00:45,294 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 21:00:44,784 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 22:00:43,975 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-15 23:00:45,103 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 00:00:43,802 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 01:00:44,195 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 02:00:43,773 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 03:00:43,355 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 04:00:44,900 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 05:00:44,447 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 06:00:43,067 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 07:00:44,243 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 08:00:45,498 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 09:00:44,242 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 10:00:45,710 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 11:00:45,591 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 12:00:44,456 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 13:00:44,594 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 14:00:45,211 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 15:00:45,083 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 16:00:45,385 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 17:00:45,559 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 18:00:53,167 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 19:00:45,798 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 20:00:47,271 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 21:00:45,181 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 22:00:45,151 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-16 23:00:44,771 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 00:00:45,163 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 01:00:43,720 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 02:00:45,994 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 03:00:44,867 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 04:00:45,873 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 05:00:45,512 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 06:00:44,891 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 07:00:45,124 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 08:00:45,522 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 09:00:46,962 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 10:00:46,590 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 11:00:46,120 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 12:00:45,890 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 13:00:12,652 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f284989fcd0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-17 13:00:12,654 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f284989fcd0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-17 14:00:46,350 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 15:00:45,649 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 16:00:14,336 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f58c80e7820>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-17 16:00:14,338 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f58c80e7820>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-17 17:00:49,735 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 18:00:13,387 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb37ff73760>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-17 18:00:13,389 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb37ff73760>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-17 19:00:47,653 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 20:00:47,277 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 21:00:47,103 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 22:00:51,813 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-17 23:00:46,361 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 00:00:47,058 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 01:00:45,780 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 02:00:12,949 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f932fc3b9d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 02:00:12,951 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f932fc3b9d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 03:00:46,089 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 04:00:48,619 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 05:00:51,686 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 06:00:12,315 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2e61327a60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 06:00:12,318 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2e61327a60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 07:00:46,717 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 08:00:52,360 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 09:00:48,232 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 10:00:52,680 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 11:00:13,828 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f076b66f2b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 11:00:13,829 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f076b66f2b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 12:00:48,845 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 13:00:47,067 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 14:00:12,866 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f45c0ff7400>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 14:00:12,877 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f45c0ff7400>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 15:00:47,892 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 16:00:54,445 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 17:00:48,995 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 18:00:48,182 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 19:00:47,500 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 20:00:47,318 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 21:00:13,671 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5b5fe9b340>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 21:00:13,675 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5b5fe9b340>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 22:00:47,926 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-18 23:00:13,062 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdc8f9638b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-18 23:00:13,063 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdc8f9638b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 00:00:13,884 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f339aa57b50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 00:00:13,887 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f339aa57b50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 01:00:57,788 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 02:00:13,411 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd00d037bb0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 02:00:13,417 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd00d037bb0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 03:00:51,106 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 04:00:17,068 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f23d48af4f0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 04:00:17,077 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f23d48af4f0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 05:00:14,534 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f489b9038e0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 05:00:14,538 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f489b9038e0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-19 06:00:50,630 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 07:00:46,516 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 08:00:49,530 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 09:00:47,409 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 10:00:47,886 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 11:00:46,969 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 12:00:46,350 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 13:00:51,127 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 14:00:51,642 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 15:00:46,796 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 16:00:47,036 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 17:00:47,311 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 18:00:46,438 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 19:00:47,161 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 20:00:46,702 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 21:00:46,836 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 22:00:48,240 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-19 23:00:46,108 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 00:00:47,858 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 01:00:46,308 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 02:00:13,951 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fea710e7610>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-20 02:00:13,953 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fea710e7610>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-20 03:00:45,722 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 04:00:50,066 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 05:00:53,743 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 06:00:46,763 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 07:00:47,484 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 08:00:49,107 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 09:00:49,259 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 10:00:14,125 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f08b53b3c40>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-20 10:00:14,128 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f08b53b3c40>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-20 11:00:51,333 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 12:00:48,367 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 13:00:49,895 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 14:00:47,393 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 15:00:13,190 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f837d4df910>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-20 15:00:13,201 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f837d4df910>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-20 16:00:48,707 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 17:00:51,688 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 18:00:51,151 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 19:00:50,299 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 20:00:49,559 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 21:00:47,403 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 22:00:48,393 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-20 23:00:49,241 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 00:00:13,049 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f20cc5f6f50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 00:00:13,052 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f20cc5f6f50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 01:00:53,482 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 02:00:47,789 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 03:00:47,607 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 04:00:56,857 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 05:00:14,643 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb064bd7280>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 05:00:14,648 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb064bd7280>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 06:00:51,529 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 07:00:15,004 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff724fdfca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 07:00:15,007 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff724fdfca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 08:00:16,630 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f71eb483be0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 08:00:16,634 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f71eb483be0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 09:00:56,426 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 10:00:15,468 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f4bf300fd60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 10:00:15,471 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f4bf300fd60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 11:00:49,671 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 12:00:55,585 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 13:00:56,470 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 14:00:57,409 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 15:00:55,640 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 16:00:57,170 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 17:00:52,301 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 18:00:52,866 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 19:00:13,356 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5541fffb50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 19:00:13,362 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5541fffb50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 20:00:14,537 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fee28a5bb20>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 20:00:14,540 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fee28a5bb20>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 21:00:55,352 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 22:00:50,524 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-21 23:00:13,031 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f57c0fbf610>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-21 23:00:13,036 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f57c0fbf610>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 00:00:54,682 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 01:00:14,168 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff2b7ca3a00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 01:00:14,172 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff2b7ca3a00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 02:00:54,535 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 03:00:49,031 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 04:00:16,828 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2a12197c10>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 04:00:16,830 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2a12197c10>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 05:00:56,781 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 06:00:49,670 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 07:00:51,410 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 08:00:50,270 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 09:00:58,960 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 10:00:52,212 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 11:00:51,297 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 12:00:58,310 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 13:00:54,486 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 14:00:55,549 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 15:00:56,246 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 16:00:13,933 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f590f33ba90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 16:00:13,937 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f590f33ba90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 17:00:14,795 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f4835cf7a30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 17:00:14,805 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f4835cf7a30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 18:00:52,730 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 19:00:14,938 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fbdbd47faf0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 19:00:14,941 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fbdbd47faf0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 20:00:50,533 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-22 21:00:14,244 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff4850b7ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 21:00:14,247 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff4850b7ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 22:00:14,385 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fcd72df77c0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 22:00:14,388 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fcd72df77c0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-22 23:00:55,904 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 00:00:14,113 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f445d2c39d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 00:00:14,151 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f445d2c39d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 01:01:05,670 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 02:00:14,783 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff95a75ba90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 02:00:14,785 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff95a75ba90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 03:00:51,130 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 04:01:02,728 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 05:00:57,592 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 06:00:55,245 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 07:00:14,635 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f772c52bd00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 07:00:14,638 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f772c52bd00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 08:00:16,568 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fce1f66fdf0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 08:00:16,570 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fce1f66fdf0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 09:00:52,621 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 10:00:57,093 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 11:00:54,339 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 12:00:54,714 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 13:00:14,244 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe54a8bb700>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 13:00:14,247 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe54a8bb700>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-23 14:00:53,573 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 15:00:57,524 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 16:00:50,630 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 17:00:52,206 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 18:01:06,727 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 19:00:54,166 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 20:01:21,028 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 21:02:04,008 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 22:00:55,890 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-23 23:00:51,351 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 00:00:55,525 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 01:00:55,527 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 02:00:13,431 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f1bc27a7940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 02:00:13,435 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f1bc27a7940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 03:00:57,284 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 04:00:56,482 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 05:00:53,280 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 06:00:57,870 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 07:00:53,658 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 08:00:55,343 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 09:00:11,960 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f987ad5b9d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 09:00:11,961 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f987ad5b9d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 10:00:59,202 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 11:00:56,041 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 12:00:55,526 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 13:00:14,791 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb3af433c70>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 13:00:14,796 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb3af433c70>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 14:00:55,132 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 15:00:59,058 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 16:01:05,728 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 17:00:55,697 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 18:00:14,522 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f1d16993520>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 18:00:14,524 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f1d16993520>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 19:00:57,638 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-24 20:00:15,010 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f092904b850>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 20:00:15,016 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f092904b850>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 21:00:14,263 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd6edf97640>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 21:00:14,268 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd6edf97640>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 22:00:15,457 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2e1b6bfa00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 22:00:15,479 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2e1b6bfa00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-24 23:00:58,391 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 00:00:17,193 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fbcbfc839d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 00:00:17,196 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fbcbfc839d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 01:01:06,420 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 02:00:15,140 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdf33217a60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 02:00:15,144 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdf33217a60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 03:00:56,587 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 04:00:17,240 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2b363ef970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 04:00:17,243 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f2b363ef970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 05:00:15,435 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f43229c3b50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 05:00:15,442 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f43229c3b50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 06:01:10,158 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 07:00:54,199 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 08:00:14,263 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdbf3857820>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 08:00:14,266 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdbf3857820>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 09:00:16,772 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3350eab9d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 09:00:16,775 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3350eab9d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 10:00:55,767 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 11:00:14,262 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc5c726f7c0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 11:00:14,268 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc5c726f7c0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 12:00:13,702 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb6cef7b940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 12:00:13,705 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb6cef7b940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 13:00:13,669 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fddcf303d00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 13:00:13,672 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fddcf303d00>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 14:00:14,163 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0a54dd39d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 14:00:14,169 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0a54dd39d0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 15:00:52,296 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 16:00:14,431 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f572cae3ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 16:00:14,433 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f572cae3ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 17:00:56,013 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 18:00:14,640 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff83c66f550>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 18:00:14,642 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ff83c66f550>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 19:00:52,488 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 20:00:16,509 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0afd8d78b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 20:00:16,511 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0afd8d78b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 21:00:16,306 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fbd3b0d3b80>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 21:00:16,312 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fbd3b0d3b80>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 22:00:59,287 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-25 23:00:13,274 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9210887970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-25 23:00:13,278 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9210887970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 00:00:56,376 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 01:00:14,743 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f123b517760>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 01:00:14,746 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f123b517760>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 02:00:15,349 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0f496e3820>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 02:00:15,352 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0f496e3820>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 03:00:58,152 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 04:01:04,867 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 05:00:17,341 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb3265a3bb0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 05:00:17,344 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fb3265a3bb0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 06:01:00,363 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 07:00:15,900 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc58391b940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 07:00:15,903 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc58391b940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 08:00:57,352 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 09:01:00,490 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 10:00:14,711 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5b61ba7940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 10:00:14,716 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5b61ba7940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 11:01:00,226 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 12:01:07,260 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 13:00:15,385 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd5e96f7880>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 13:00:15,390 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd5e96f7880>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 14:01:00,858 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 15:00:56,562 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 16:00:14,329 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f8968093af0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 16:00:14,332 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f8968093af0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 17:00:58,230 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 18:01:05,228 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 19:00:15,472 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd0d1b0b850>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 19:00:15,475 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd0d1b0b850>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 20:01:02,852 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 21:00:53,151 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 22:01:03,754 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-26 23:00:15,035 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0f2193f940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-26 23:00:15,037 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0f2193f940>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 00:01:01,018 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 01:01:03,047 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 02:00:16,288 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc145553ac0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 02:00:16,293 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fc145553ac0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 03:00:59,302 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 04:00:22,051 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f123719fa90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 04:00:22,056 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f123719fa90>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 05:01:02,820 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 06:00:59,494 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 07:00:56,823 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 08:00:57,497 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 09:00:59,896 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 10:01:02,663 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 11:01:02,398 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 12:00:58,543 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 13:00:19,883 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fea51b67be0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 13:00:19,887 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fea51b67be0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-27 14:01:05,898 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 15:00:59,900 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 16:01:08,584 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 17:00:59,532 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 18:01:03,177 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 19:01:07,269 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 20:00:57,458 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 21:00:56,625 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 22:01:02,056 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-27 23:00:58,833 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 00:00:54,872 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 01:00:59,027 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 02:00:54,493 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 03:00:53,145 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 04:01:10,944 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 05:01:00,232 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 06:00:55,383 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 07:01:00,322 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 08:00:16,177 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f440f5cbcd0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 08:00:16,183 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f440f5cbcd0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 09:01:04,142 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 10:01:24,071 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 11:00:14,364 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f438f3a7640>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 11:00:14,367 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f438f3a7640>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 12:00:17,015 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe76c5cf730>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 12:00:17,031 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fe76c5cf730>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 13:00:15,835 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5ce720f8b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 13:00:15,839 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5ce720f8b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 14:00:16,986 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f96e2633d60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 14:00:16,993 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f96e2633d60>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 15:00:17,254 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f02bad63ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 15:00:17,259 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f02bad63ca0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 16:00:16,870 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fa7d5267b50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 16:00:16,887 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fa7d5267b50>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 17:01:11,422 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 18:00:16,305 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3e239c77c0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 18:00:16,308 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3e239c77c0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 19:01:10,662 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 20:01:10,900 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-28 21:00:18,876 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3c9fc078b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 21:00:18,881 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3c9fc078b0>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 22:00:17,434 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7cd025b970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 22:00:17,476 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7cd025b970>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-28 23:01:23,836 - ERROR - ETL Process for Workly failed: ('invalid_grant: Invalid JWT Signature.', {'error': 'invalid_grant', 'error_description': 'Invalid JWT Signature.'})
2025-01-29 00:00:18,415 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f893d253b20>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2025-01-29 00:00:18,431 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f893d253b20>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
